PasswordReset/PasswordReset/settings.py

```diff
@@ -0,0 +1,198 @@
+# -*- coding: utf-8 -*-
+"""
+Django settings for PasswordReset project.
+Generated by 'django-admin startproject' using Django 1.11.1.
+For more information on this file, see
+https://docs.djangoproject.com/en/1.11/topics/settings/
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.11/ref/settings/
+"""
+
+import os
+import app.providers
+import environ
+env = environ.Env()
+environ.Env.read_env()
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
```

Comment:
does the comment above (don't run with debug turned on in production) apply here?

Reply:
it should apply here :p Will make the change.